### ISSUE TITLE
Endpoint location is not relative

### ISSUE THREAD

**Issue by grishace** [Nhowka/Elmish.Bridge#12]
When relative endpoint location is specified in `Program.withBridge` call, it is not relative during the runtime.

Consider the scenario, when app is developed locally as standalone - i.e. all routes are root-based, but deployed to IIS as a virtual app under some folder.  

[Here](https://github.com/Nhowka/Elmish.Bridge/blob/master/src/Client/Library.fs#L51)
the endpoint location with the existing path info is being replaced with  the one passed with the bridge configuration.

Maybe it would be better to do this instead?
`url.pathname <- url.pathname  + this.path`

**Comment by Nhowka** [comment]
That's not always desirable because it could be used with a different path that gives information to Elmish when using the navigator module, making the path to the socket wrong on those situations. I just pushed an update to give it more control about using a relative or even a real absolute path if needed. The [README](https://github.com/Nhowka/Elmish.Bridge#configuring-the-endpoint) was updated as well.
